### ISSUE TITLE
Slider cannot navigate among states in Chrome 52

### ISSUE THREAD

**Issue by mocheng** [zalmoxisus/redux-devtools-extension#170]
Chrome version: Version 52.0.2743.82 (64-bit)
Mac El Capitan 10.11.6 (15G31)
Redux devtools extension version: 2.3.0

In the bottom Redux Panel of Chrome dev tools, the slider cannot navigate state to make the page re-render.
![image](https://cloud.githubusercontent.com/assets/239291/17167652/d264cb7e-5410-11e6-8e1b-6ce7ec61f7ac.png)

However, the slider in popup triggered by bookmark bar works.
![image](https://cloud.githubusercontent.com/assets/239291/17167678/fb500ea4-5410-11e6-80fa-8dcc9c981d40.png)

**Comment by zalmoxisus** [comment]
Thanks for the report. Can you reproduce it with [our demos](https://github.com/zalmoxisus/redux-devtools-extension#demo)? If not, please provide a repo or jsfiddle to reproduce that.

**Comment by mocheng** [comment]
Interesting. The demos all work fine. I'll follow up on what breaks in my case.

**Comment by mocheng** [comment]
@zalmoxisus I managed to locate the root cause of the bug. In my project, there is one thunk action that will dispatch an plain-object action with big array payload. If the I manually reduce the big array payload, the bug disappears. 

But, while tweak example project to dispatch big payload action, this bug cannot be reproduced. Sorry that my project is private cannot be exposed, but it looks this extension doesn't handle big payload action with `redux-thunk` middleware.

**Comment by zalmoxisus** [comment]
@mocheng, don't see how this could be related to the slider monitor, but you can check whether it is the problem by stripping the action's payload at all:
```js
const store = Redux.createStore(reducer, window.devToolsExtension && window.devToolsExtension({
  actionsFilter: (action) => ({ type: action.type })
}));
```
You could also use it to strip only that action's key.

**Comment by mocheng** [comment]
@zalmoxisus If I filter that payload, that application would not work at all. 

I'm not familiar with extension development, any quick way to enable me do debugging or adding console.log to locate the buggy code?

**Comment by mocheng** [comment]
@zalmoxisus  Finally I got it reproduced by tweaking example code. Please check this out https://github.com/mocheng/redux-devtools-extension/tree/reproduce-issue-170/examples/counter

In the forked repo branch `reproduce-issue-170`, the example is `examples/counter`. After launching the web page, click `increment async` button several times, then check the slider. While sliding the slider, the counter value is not updated.

Basically, the change is just filling Redux state with some big data, which can trigger the issue.
https://github.com/zalmoxisus/redux-devtools-extension/pull/171/files

**Comment by zalmoxisus** [comment]
@mocheng, thanks a lot for the fork and details!

I replicated the problem with the release from Chrome store, but it works well with the current development version. So, should be fixed already (probably by [88046ffb752695e3b1feca42fb7231e5a2331c20](https://github.com/zalmoxisus/remotedev-app/commit/88046ffb752695e3b1feca42fb7231e5a2331c20)). I'll give it another try before publishing (most likely at the end of the week).

**Comment by zalmoxisus** [comment]
Should be fixed in `v2.4.0`.

**Comment by mocheng** [comment]
It works now. Thanks.
